pkgs/development/python-modules/cpyparsing/default.nix

```diff
@@ -0,0 +1,20 @@
+{lib, buildPythonPackage, fetchPypi, cython}:
+
+buildPythonPackage rec {
+  pname = "cPyparsing";
+  version = "2.4.5.0.1.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1kcr8v9yx9whbdn7c7rnfg4b6bk3agpa1wblhr2y6n93r8a4bxwy";
+  };
+
+  nativeBuildInputs = [ cython ];
+
```

Comment:
There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.

```

----------------------------------------------------------------------
Ran 0 tests in 0.000s
```